### DISCUSSION TITLE
I can't see the SD.NEXT images-browser anymore.

### DISCUSSION THREAD

**Discussion by sevenlucas7** [vladmandic/sdnext#1360] [Q&A]
I have a machine running Windows 11 with an NVIDIA RTX 4080 laptop GPU. When running webui.bat, I encountered the following error message, and the images-browser in SD.NEXT is not loading or visible. Can you please advise on how to resolve this issue?
<img width="856" alt="image" src="https://github.com/vladmandic/automatic/assets/131904649/75c10f1b-9165-4ef4-9eac-4b0cbf11acb7">

**Comment by vladmandic** [comment]
just to double-check, does the issue persists if you restart the app?

**Reply by sevenlucas7** [reply]
Yes, the issue still persists.

**Reply by vladmandic** [reply]
is this a new install? because i cannot reproduce and looking at the code, only way this should happen is if image browser database is corrupt so get_last_default_tab returns None.

in any case, should be reported to image browser repository.

**Reply by sevenlucas7** [reply]
Yes, it's a new installation. I didn't encounter this issue on my MacBook, only on this Windows 11 machine.

**Reply by vladmandic** [reply]
i would delete `extensions-builtin/stable-diffusion-webui-images-browser` and let it be recreated by installer.

**Reply by sevenlucas7** [reply]
ok`! thx
